### PR TITLE
Maya: Correct Alembic export defaults AY-5273

### PR DESCRIPTION
**Pull request by tokejepsen**
## Changelog Description
Missing `writeUVs` on the Alembic extraction.

## Testing notes:
1. Remove any overrides on `project_settings/maya/publish/ExtractAlembic`
2. Publish pointcache or animation.
3. Validate published geometry has UVs.

### CODE REVIEW

**Comment by moonyuet** [comment]
I slightly look at the code and I compared with the `schema_maya_publish.json` and it doesn't have the `WriteUVs` flag (they have `WriteUVSets` flag instead). Are we gonna add or change the name of that specific flag in `schema_maya_publish.json`? 
I am asking this as `WriteUVs `doesn't show up in my setting
image

image

**Comment by BigRoy** [comment]
> I slightly look at the code and I compared with the `schema_maya_publish.json` and it doesn't have the `WriteUVs` flag (they have `WriteUVSets` flag instead). Are we gonna add or change the name of that specific flag in `schema_maya_publish.json`? image I am asking this as `WriteUVs `doesn't show up in my setting image

Note that I believe `writeUVSets` and `writeUVs` is a different flag on the alembic exporter. The first meaning "export multiple UV sets".

See both present here:
- https://github.com/alembic/alembic/blob/master/maya/AbcExport/AbcExport.cpp#L389
- https://github.com/alembic/alembic/blob/master/maya/AbcExport/AbcExport.cpp#L414

**Comment by moonyuet** [comment]
> > I slightly look at the code and I compared with the `schema_maya_publish.json` and it doesn't have the `WriteUVs` flag (they have `WriteUVSets` flag instead). Are we gonna add or change the name of that specific flag in `schema_maya_publish.json`? image I am asking this as `WriteUVs `doesn't show up in my setting image
> 
> Note that I believe `writeUVSets` and `writeUVs` is a different flag on the alembic exporter. The first meaning "export multiple UV sets" - the other meaning 'write any UV set'. If only the latter is enabled then only the current UV set would be exported.
> 
> See both present here:
> 
> * https://github.com/alembic/alembic/blob/master/maya/AbcExport/AbcExport.cpp#L389
> * https://github.com/alembic/alembic/blob/master/maya/AbcExport/AbcExport.cpp#L414
> 
> Not entirely sure, but I believe that is the case.

We need to add `WriteUVs` into the `schema_maya_publish.json` then, if both `WriteUVs` and `WriteUVSets` existed as job arguments.

**Comment by BigRoy** [comment]
> We need to add `WriteUVs` into the `schema_maya_publish.json` then, if both `WriteUVs` and `WriteUVSets` existed as job arguments.

Thanks - I edited my previous comment seems I misremembered slightly. We need both `uvWrite` and `writeUVSets`. And indeed `writeUVs` seems to be invalid/typo.

File: openpype/settings/defaults/project_settings/maya.json [hunk 1]
```diff
@@ -1114,7 +1114,8 @@
             "flags": [
                 "stripNamespaces",
                 "writeNormals",
-                "worldSpace"
+                "worldSpace",
+                "writeUVs"
```
Comment (tokejepsen) [inline openpype/settings/defaults/project_settings/maya.json:1118]:
```suggestion
                "uvWrite"
```

**Review by moonyuet — APPROVED** [review]
Tested successfully in Maya
image

**Comment by BigRoy** [comment]
> Tested successfully in Maya image

Would be good - that aside of the publish passing you also confirm in the test that the resulting publish contains UVs.

**Comment by moonyuet** [comment]
> > Tested successfully in Maya image
> 
> Would be good - that aside of the publish passing you also confirm in the test that the resulting publish contains UVs.

yes it has the UVs.
image

**Review by BigRoy — APPROVED** [review]
Thanks!

**Pull request merged**